packages/www/hooks/use-api.tsx

```diff
@@ -823,6 +824,75 @@ const makeContext = (state: ApiState, setState) => {
       return res;
     },
 
+    async createAsset(params): Promise<Asset> {
+      const [res, asset] = await context.fetch(`/asset/import`, {
+        method: "POST",
+        body: JSON.stringify(params),
+        headers: {
+          "content-type": "application/json",
+        },
+      });
+
+      if (res.status !== 201) {
+        throw new Error(asset.errors.join(", "));
+      }
+      return asset;
+    },
+
+    async getAssets(
+      userId: string,
+      opts?: {
+        filters?: Array<{ id: string; value: string | object }>;
+        limit?: number | string;
+        cursor?: string;
+        order?: string;
+        active?: boolean;
+        count?: boolean;
+      }
+    ): Promise<[Asset[], string, number]> {
+      const filters = opts?.filters ? JSON.stringify(opts?.filters) : undefined;
+      const [res, assets] = await context.fetch(
+        `/asset?${qs.stringify({
+          userId,
+          filters,
+          active: opts?.active,
+          order: opts?.order,
+          limit: opts?.limit,
+          cursor: opts?.cursor,
+          count: opts?.count,
+          streamsonly: 1,
+        })}`
+      );
+      if (res.status !== 200) {
+        throw new Error(assets);
+      }
+      const nextCursor = getCursor(res.headers.get("link"));
+      const count = res.headers.get("X-Total-Count");
+      return [assets, nextCursor, count];
+    },
+
+    async deleteAsset(id: string): Promise<void> {
```

Comment:
Just a note for us: the delete asset flow is only partially implemented on the backend, missing https://github.com/livepeer/task-runner/issues/8 and https://github.com/livepeer/task-runner/issues/9